frontend/main-app/src/components/PlaceHolderText/PlaceHolderText.tsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from "react";
+import { Heading } from "@digdir/design-system-react";
+
+export const PlaceholderText = (): React.JSX.Element => {
+  return (
+    <>
+      <Heading level={1}>Welcome to the Form Factory</Heading>
+      <p>
```

Comment:
I know it is just a placeholder text, but to get used to the Designsystem, I suggest that we maybe can use the ```<Paragraph />``` component [from Designsystemet](https://storybook.designsystemet.no/?path=/docs/felles-typography-paragraph--docs) instead of a ```<p/>```?😄

Reply:
Agreed - added 👍